cmd/oci-image-validate/main.go

```diff
@@ -104,7 +104,7 @@ func newValidateCmd(stdout, stderr *log.Logger) *cobra.Command {
 func (v *validateCmd) Run(cmd *cobra.Command, args []string) {
 	if v.version {
 		v.stdout.Printf("commit: %s", gitCommit)
-		v.stdout.Printf("spec: %s", specs.Version)
+		v.stdout.Printf("spec: %s", version.Version)
```

Comment:
This isn't correct. We should output separate lines for what `spec` versions we support, and the version of the actual tool.